client.go

```diff
@@ -49,13 +56,19 @@ func Dial(s oauth2.TokenSource) (*Client, error) {
 		c.Account = &a[0]
 	}
 
+	ca, err := c.GetCryptoAccounts()
+	if len(ca) > 0 {
+		c.CryptoAccount = &ca[0]
+	}
+
 	return c, err
 }
 
 // GetAndDecode retrieves from the endpoint and unmarshals resulting json into
 // the provided destination interface, which must be a pointer.
 func (c *Client) GetAndDecode(url string, dest interface{}) error {
 	req, err := http.NewRequest("GET", url, nil)
+	req.Header.Set("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36")
```

Comment:
Are things not working without mocking a browser user agent? I'd probably prefer to set something custom to our client, since that might help Robinhood understand the demand for continued support of API clients. If they're blocking requests by user agent, though, I'm not sure how I feel about pretending to be a browser.

Reply:
thank you for reviewing this. I will get rid of the user agent.

Reply:
removed it.